lua/lazyvim/keymaps.lua

```diff
@@ -0,0 +1,673 @@
+        to_pipe = "<leader>cp",
+        from_pipe = "<leader>cP",
+      },
+      lean = {
+        abbreviations_leader = "\\",
+      },
+      markdown = {
+        markdown_preview_toggle = "<leader>cp",
+        render_markdown_toggle = "<leader>um",
+      },
+      python = {
+        debug_method = "<leader>dPt",
+        debug_class = "<leader>dPc",
+        select_virtual_env = "<leader>cv",
+      },
+      r = {
+        send = "<Enter>",
+        send_all = "<localleader>a",
+        send_between_marks = "<localleader>b",
+        send_chunks = "<localleader>c",
+        send_functions = "<localleader>f",
+        send_goto = "<localleader>g",
+        send_install = "<localleader>i",
+        send_knit = "<localleader>k",
+        send_paragraph = "<localleader>p",
+        send_quarto = "<localleader>q",
+        send_general = "<localleader>r",
+        send_split_or_send = "<localleader>s",
+        send_terminal = "<localleader>t",
+        send_view = "<localleader>v",
+      },
+      rust = {
+        code_action = "<leader>cR",
+        debuggables = "<leader>dr",
+        show_crate_documentation = "K",
+      },
+      scala = {
+        metals_commands = "<leader>me",
+        metals_compile_cascade = "<leader>mc",
+      },
+      sql = {
+        toggle_dbui = "<leader>D",
+      },
+      tex = {
+        prefix = "<localLeader>l",
+        vimtex_docs = "<Leader>K",
+      },
+      typescript = {
+        go_to_source_definition = "gD",
+      },
+    },
+    ui = {
+      edgy = {
+        toggle = "<leader>ue",
+        select_window = "<leader>uE",
+      },
+      mini_animate = {
+        toggle = "<leader>ua",
+      },
+      treesitter_context = {
+        toggle = "<leader>ut",
+      },
+    },
+    util = {
+      chezmoi = {
+        pick_chezmoi = "<leader>sz",
+        select = "<CR>",
+        key = "c",
+      },
+      octo = {
+        issue_list = "<leader>gi",
+        issue_search = "<leader>gI",
+        pr_list = "<leader>gp",
+        pr_search = "<leader>gP",
+        repo_list = "<leader>gr",
+        search = "<leader>gS",
+        assignee = "<leader>a",
+        comment_code = "<leader>c",
+        label = "<leader>l",
+        issue = "<leader>i",
+        react = "<leader>r",
+        pr = "<leader>p",
+        review = "<leader>v",
+        insert_at = "@",
+        insert_hashtag = "#",
+      },
+      rest = {
+        kulala = {
+          prefix = "<leader>R",
+          send_request = "<leader>Rs",
+          toggle_view = "<leader>Rt",
+          jump_to_previous_request = "<leader>Rp",
+          jump_to_next_request = "<leader>Rn",
+        },
+      },
+    },
+    vscode = {
+      find = "<leader><space>",
+      find_in_files = "<leader>/",
+      go_to_symbol = "<leader>ss",
+    },
+  },
+}
+
+---@return LazyVimKeymaps
+M.get_keymaps = function()
+  if keymaps == nil then
+    keymaps = vim.tbl_deep_extend("force", {}, M.default_keymaps, vim.g.lazyvim_keymaps)
```

Comment:
```suggestion
    keymaps = vim.tbl_deep_extend("force", {}, M.default_keymaps, vim.g.lazyvim_keymaps or {})
```
Here you need a check for empty table in case `vim.g.lazyvim_keymaps` is not defined. I tried your PR locally and Neovim crashed with error about `vim.tbl_deep_extend` 2nd argument expecting a table but got `nil`.

Reply:
You can test by running in your local forked repo at the root of the repo `./scripts/test`.

Reply:
Thank you, I really appreciate it, already passed locally. But I remembered I had included this don't know why and when I removed it 😆